front/src/views/recovery-password/RecoveryPassword.html

```diff
@@ -0,0 +1,162 @@
+          </div>
+        </v-stepper-step>
+      </v-stepper-header>
+      <v-stepper-items>
+        <v-stepper-content step="1">
+          <v-card color="#F6F8FB">
+            <div class="text-center text-normal">
+              {{$t('recoverypassword.Entuemail')}}
+            </div>
+            <v-form ref="form" v-model="valid">
+              <div class="input-email">
+                <mmt-text-input
+                  class="input-field"
+                  mmt-label="E-mail"
+                  rounded
+                  filled
+                  single-line
+                  dense
+                  :placeholder="$t('recoverypassword.emailPlh')"
+                  v-model="email"
+                  :rules="[rules.isEmpty, rules.validEmail]"
+                  :error-messages="serverError"
+                  @input="onInput"
+                  required
+                />
+              </div>
+              <v-row cols="2" justify="center">
+                <v-col justify="center">
+                  <v-btn
+                    class="mmt-btn btn-step-1"
+                    rounded
+                    color="primary"
+                    :disabled="!valid"
+                    @click="recoveryPassword"
+                    >{{$t('recoverypassword.recover')}}
+                  </v-btn>
+                </v-col>
+                <v-col justify="center">
+                  <v-btn
+                    rounded
+                    outlined
+                    color="accent lighten-1"
+                    class="mmt-btn btn-step-1"
+                    dense
+                    @click="$router.push('/');"
+                    >{{$t('recoverypassword.back')}}
+                  </v-btn>
+                </v-col>
+              </v-row>
+            </v-form>
+            <div class="text-center text-normal">
+              {{$t('recoverypassword.note')}}
+            </div>
+          </v-card>
+        </v-stepper-content>
+        <v-stepper-content step="2">
+          <v-card color="#F6F8FB">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.excelentwrk')}}
+          </div>
+          <v-form ref="form" v-model="valid">
+            <div class="input-firstPass">
+              <mmt-text-input
+                :placeholder="$t('recoverypassword.createpass')"
+                :mmt-label="$t('recoverypassword.password')"
+                class="input-field"
+                rounded
+                filled
+                dense
+                :type="showfirstPass ? 'text' : 'password'"
+                :append-icon="showfirstPass ? 'mdi-eye' : 'mdi-eye-off'"
+                @click:append="showfirstPass = !showfirstPass"
+                :rules="[rules.isEmpty, rules.minPassLen]"
+                v-model="firstPass"
+                required
+              />
+            </div>
+            <div class="input-secondPass">
+              <mmt-text-input
```

Comment:
btw, you can extract this password with eye icon which can flip from type=text to type=password into a separate component and don't clutter or overload pages with repetitive elements

Reply:
I'll think about it)